.github/workflows/emscripten.yml

```diff
@@ -9,6 +9,18 @@ on:
     branches:
       - master
       - v1.**
+  # Make it possible to upload wheels manually if needed
+  workflow_dispatch:
+    inputs:
+      push_wheels:
+        description: >
+          'Push wheels to Anaconda if "true". Default is "false". Warning: this will overwrite existing wheels.'
+        required: false
+        default: "false"
+  # Upload wheels to Anaconda on a schedule
+  schedule:
+    # Run at 0300 hours on days 3 and 28 of the month
+    - cron: "0 3 3,28 * *"
```

Comment:
I had to resort to this schedule because I could not find a cleaner way to represent "once every 25 days" with a CRON job. There are some insights in [this thread](https://unix.stackexchange.com/questions/442792/how-to-schedule-cronjob-for-every-45-days), but I think keeping it simple is better?

Reply:
Sure, that works too. But `3,17` doesn't sound equally spaced – did you mean `3,27`?

Reply:
No, I meant `3,17`. That's 14 days in between from 3 to 17, and 17th to 3rd of the next month is also about 2 weeks.